scaleway/provider.go

```diff
@@ -225,6 +225,7 @@ func Provider() terraform.ResourceProvider {
 			"scaleway_instance_server":         dataSourceScalewayInstanceServer(),
 			"scaleway_instance_image":          dataSourceScalewayInstanceImage(),
 			"scaleway_instance_volume":         dataSourceScalewayInstanceVolume(),
+			"scaleway_baremetal_offer_beta":    dataSourceScalewayBaremetalOfferBeta(),
```

Comment:
nit: move it (in the map) with other baremetal resources?

Reply:
this is the data sources map

Reply:
:palmface: